docs/pages/router/static-rendering/root-html.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: Root HTML
+description:
+hideFromSearch: true
+---
+
+> **warning** This guide requires `expo@49.0.0-alpha.3` or above. Everything listed here is still experimental. You may need to use [Expo CLI on `main`](https://github.com/expo/expo/tree/main/packages/%40expo/cli#contributing) to use these features.
+
+When you statically render an Expo website (`EXPO_USE_STATIC=1 yarn expo start` cite needed), the root HTML element for each page can be customized by creating an **app/+html.js** file that exports a default HTML component.
```

Comment:
```suggestion
When you statically render an Expo website, the root HTML element for each page can be customized by creating an **app/+html.js** file that exports a default HTML component.
```
The static rendering is now enabled by setting `web.output: 'static'` in the app.json